scripts/build.py

```diff
@@ -1177,6 +1177,40 @@ def compile_typescript_files(project_dir):
     subprocess.check_call(cmd, shell=True)
 
 
+def compile_typescript_files_in_watch_mode(project_dir):
```

Comment:
compile_typescript_files_continuously

and then, in the docstring, you can explain what "continuously" means.

Reply:
Done.